build/ProjectReunion-BuildFoundation.yml

```diff
@@ -263,6 +296,22 @@ jobs:
       - BuildAndTestMRT
     condition: eq(variables['Build.SourceBranch'], 'refs/heads/develop')
 
+# Arm64 uses a different VM type than x64, thus requiring them to be in different
+# pools, hence, we spilt the two scenarios into separate jobs.
+- template: AzurePipelinesTemplates\WindowsAppSDK-RunTestsInPipeline-Job.yml
+  parameters:
+    jobName: PipelineTestsX64
+    isArm64Platfrom: false
+    dependsOn:
+      - Build
```

Comment:
Should depend on all of these job below
      - Build
      - BuildAnyCPU
      - BuildAndTestMRT

Reply:
I was considering each of the jobs and found that the only job testing has a dependency on is the Build job.

Reply:
But if in the future, we want to test MRT core in these pipeline tests, it would need to depend on BuildAndTestMRT.

Reply:
That's a good point. This PR should prep for MRT testing as well.

Reply:
Ideally all those jobs will be in a staged called "Build" and a "Test" stage would depend on the "Build"stage.
It would require some refactoring to move all these jobs into stages.

Stretch goal for you :)